scripts/release/controller/setup.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
I removed the checks for Ubuntu 16 since we're checking against an 18.0.4 image.  Should we still be testing that version?

I removed other lines checking for the pre-existence of something on the fs since the build spins up a fresh instance every time it's run.  I felt it'd be better to run it as a blank state so there's no remaining state from a previous build.